src/filter/exceptions.filter.ts

```diff
@@ -0,0 +1,64 @@
+import {
+  ArgumentsHost,
+  BadRequestException,
+  Catch,
+  ExceptionFilter,
+  HttpStatus,
+} from "@nestjs/common";
+
+import { ErrorResponse } from "../dto/error-response.dto";
+import { ExtendedError } from "../dto";
+import { Response } from "express";
+import { InvalidCredentialsError } from "../exception/invalid-credentials.error";
+import { UserExistsError } from "../exception/user-exists.error";
+import { InvalidResetPasswordRequestError } from "../exception/invalid-reset-password-request.error";
+import { InvalidRefreshTokenError } from "../exception/invalid-refresh-token.error";
+import { InvalidOtpError } from "../exception/invalid-otp.error";
+import { UserNotFoundError } from "../exception/user-not-found.error";
+
+@Catch(Error)
+export class ExceptionsFilter implements ExceptionFilter {
```

Comment:
Expection classes will have status information. Can we review the example and update it according to the incoming data? 

https://docs.nestjs.com/exception-filters#exception-filters-1